templates/keepalived/0/docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+version: '2'
+services:
+  keepalived-master:
+    restart: always
+    image: arcts/keepalived:1.1.0
+    network_mode: host
+    cap_add:
+      - NET_ADMIN
+    environment:
+      KEEPALIVED_AUTOCONF: true
+      KEEPALIVED_STATE: MASTER
+      KEEPALIVED_INTERFACE: ${interface}
+      KEEPALIVED_VIRTUAL_ROUTER_ID: ${router_id}
+      KEEPALIVED_UNICAST_SRC_IP: ${master_ip}
+      KEEPALIVED_UNICAST_PEER_0: ${backup_ip}
+      KEEPALIVED_TRACK_INTERFACE_1: ${interface}
+      KEEPALIVED_VIRTUAL_IPADDRESS_1: "\"${virtual_ip}\""
+    labels:
+      io.rancher.scheduler.affinity:host_label: ${host_label}=${master_label}
```

Comment:
What do you think to add optional sysctl update, with go templating?? 
Rename docker-compose.yml to docker-compose.yml.tpl and add
```
+    {{- if eq .Values.UPDATE_SYSCTL "true"}}
+      io.rancher.sidekicks: keepalived-sysctl
+    {{- end}}
```